docs/developer/app-store/apps/adyen.mdx

```diff
@@ -0,0 +1,355 @@
+---
+title: Adyen
+sidebar_position: 2
+---
+
+# Adyen
+
+## Overview
+
+Adyen App is a payment integration app that allows merchants using the Saleor e-commerce platform to accept online payments from customers using Adyen as their payment processor. In addition to processing payments, the Saleor App Payment Adyen provides merchants with tools for managing refunds and chargebacks.
+
+:::caution
+
+To configure the Adyen App, you must have an account with [Adyen](https://www.adyen.com).
+
+:::
+
+The Adyen App allows for integrations with [Adyen Web Drop-in](https://docs.adyen.com/online-payments/web-drop-in/additional-use-cases/advanced-flow), [Adyen iOS Drop-in](https://docs.adyen.com/online-payments/ios/drop-in/additional-use-cases/advanced-flow), [Adyen Android Drop-in](https://docs.adyen.com/online-payments/android/drop-in), and [Adyen React Native Drop-in](https://docs.adyen.com/online-payments/react-native/drop-in). It uses the Adyen Drop-in Advanced Flow.
+
+:::info
+
+Adyen App uses Adyen [Checkout API v70](https://docs.adyen.com/api-explorer/#/CheckoutService/v70/overview) and [Management API v1](https://docs.adyen.com/api-explorer/#/ManagementService/v1/overview).
```

Comment:
Are we sure we want to link to specific API version in docs and not app's README? 🤔 
It might get out of date fast - once we update `@adyen/node-api-library`

Reply:
I think we do. That gives valuable information to the users and also Adyen only bumps API versions in major releases.